plugins/jobs/web_client/views/JobListWidget.js

```diff
@@ -190,9 +192,12 @@ var JobListWidget = View.extend({
     },
 
     _renderData: function () {
-        var jobs = this.collection.toArray();
+        if (!this.$('.g-main-content').length) {
+            // Do nothing until render has been called
+            return;
+        }
 
```

Comment:
render() is called at the end of initialize(), fetch() is async. I don't think renderData could be called before render().